components/get_android_app/get_android_app.jsx

```diff
@@ -3,14 +3,24 @@
 
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
-import {Link} from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import {useSafeUrl} from 'utils/url';
 import MattermostIcon from 'images/favicon/android-chrome-192x192.png';
 import Nexus6Mockup from 'images/nexus-6p-mockup.png';
 
-export default function GetAndroidApp({androidAppDownloadLink}) {
+export default function GetAndroidApp({androidAppDownloadLink, history, location}) {
+    const clickedContinueWithBrowser = (e) => {
```

Comment:
nit: `clickedContinueWithBrowser` seems like an awkward name for a handler. Perhaps `onContinue`?